src/index.html

```diff
@@ -3,12 +3,56 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
```

Comment:
Remember about attribute formatting
```suggestion
    <meta 
    	name="viewport"
```